dataclients/kubernetes/testdata/ingressV1/tls/tls-invalid-tls.yaml

```diff
@@ -0,0 +1,92 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: myapp-deployment
+  labels:
+    app: myapp
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: myapp
+  template:
+    metadata:
+      labels:
+        app: myapp
+    spec:
+      containers:
+        - name: myapp
+          image: myapp:v1
+          ports:
+            - containerPort: 80
+              name: my-port
+              protocol: TCP
+---
+apiVersion: v1
+kind: Service
+metadata:
+  labels:
+    app: myapp-deployment
+  name: myapp-service
+spec:
+  clusterIP: 10.3.190.1
+  ports:
+    - name: this-is-my-service-port-name
+      port: 8080
+      protocol: TCP
+      targetPort: my-port
+  selector:
+    app: myapp
+  type: ClusterIP
+---
+apiVersion: networking.k8s.io/v1
+kind: Ingress
+metadata:
+  labels:
+    app: myapp
+  name: myapp-ingress
+  namespace: default
+spec:
+  tls:
+  - secretName: myapp-secret
+    hosts:
+    - example.org
+  rules:
+    - host: example.org
+      http:
+        paths:
+          - backend:
+              service:
+                name: myapp-service
+                port:
+                  number: 8080
+            pathType: ImplementationSpecific
+---
+apiVersion: v1
+kind: Endpoints
+metadata:
+  labels:
+    app: myapp-deployment
+  name: myapp-service
+subsets:
+  - addresses:
+      - ip: 10.3.0.3
+        targetRef:
+          kind: Pod
+          name: myapp-deployment-6786bf95fd-fnqnq
+    ports:
+      - name: this-is-my-service-port-name
+        port: 80
+        protocol: TCP
+---
+apiVersion: v1
+kind: Secret
+metadata:
+  name: myapp-secret
+  namespace: default
+type: kubernetes.io/tls
+data:
+  tls.crt: |
```

Comment:
minor: It is not clear out of the data how it is invalid. Maybe a simple `this is invalid` string would work equally well fro the test but would make it obvious that is it not base64 encoded PEM